addons/mail/static/src/core/common/message_reaction_list.js

```diff
@@ -0,0 +1,58 @@
+/* @odoo-module */
```

Comment:
Note that there's a recent PR that make this `@odoo-module */` optional. You can remove it